Formula/tiv.rb

```diff
@@ -0,0 +1,30 @@
+class Tiv < Formula
+  desc "Display images in a terminal using block graphic characters"
+  homepage "https://github.com/stefanhaustein/TerminalImageViewer"
+  url "https://github.com/stefanhaustein/TerminalImageViewer/archive/refs/tags/v1.1.1.tar.gz"
+  sha256 "9a5f5c8688ef8db0e88dfcea6a1ae30da32268a7ab7972ff0de71955a75af0db"
+  license "Apache-2.0"
+  head "https://github.com/stefanhaustein/TerminalImageViewer.git", branch: "master"
+
+  depends_on "imagemagick"
+
+  uses_from_macos "curl" => :test
```

Comment:
How is `curl` used in the test?

Reply:
That was a relic from the olden days when I didn't know there were test fixtures! It has since been [removed](https://github.com/Homebrew/homebrew-core/pull/94168/commits/33f3f43b6f2ec20b72f2ac9b4c3ed8ebdc36014c).